Source/WarpX.H

```diff
@@ -653,11 +670,22 @@ private:
     int n_buffer = 4;
     amrex::Real const_dt = 0.5e-11;
 
+    // Load balancing
     int load_balance_int = -1;
     amrex::Vector<std::unique_ptr<amrex::MultiFab> > costs;
```

Comment:
Please add parameter descriptions `type name; //! what is my unit or how can I find it in AMReX docs` for those members, I fear we will loose track otherwise.

Reply:
Parameter descriptions added.